### PR TITLE
Bug 1176700 - Unnecessary gray padding at the bottom of a website detail page

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

**Comment by mstriemer** [comment]
This seems like it's probably a good idea. Atique had a change in #1385 that removed the scrolling which I think is what we wanted for bug 1183067 though. Is there another issue that this should be resolving?

**Comment by tsl143** [comment]
yes,for example privacy policy page.
The idea is on mobile screens, header height should be excluded from screen height to fix main body minHeight,

**Comment by tsl143** [comment]
Changed the bug number

**Comment by atiqueahmedziad** [comment]
@mstriemer r?

**Comment by tsl143** [comment]
Any updates...

File: src/media/js/elements/nav.js [hunk 1]
```diff
@@ -251,13 +251,14 @@ define('elements/nav',
         // 100% height would mess up the page since our <footer> isn't part of
         // <main>. Could be resolved if we made <footer> a part of <main>.
         var main = document.querySelector('main');
+        var minHeight = screen.height - 60;
```
Comment (mstriemer) [inline src/media/js/elements/nav.js]: I think this should actually be `window.innerHeight`

**Comment by mstriemer** [comment]
I tried this in the Chrome emulator mode set to a Nexus 6 and `screen.height` is `732` and `window.innerHeight` is `659`. I believe the stray pixels are for the software buttons at the bottom. This is probably also reproducible on Windows (and possibly Linux) since the border of the window is usually excluded from `window.innerHeight` but included in the other ways of accessing the height.

**Comment by tsl143** [comment]
@mstriemer  screen.height is consistent throughout the browsers in both linux and Windows, so can we proceed with screen.height?

**Comment by mstriemer** [comment]
I think `window.innerHeight` is the right thing to use, I'd prefer we switch to that. In the simulator (I know this isn't a real example but this is what I think `screen.height` _should_ return) I get lots of padding since the height of my screen is 900 pixels but the actual emulator is only using 540 pixels.

screenshot 2015-07-22 12 25 00

**Comment by tsl143** [comment]
@mstriemer updated the code with window.innerHeight

**Comment by mstriemer** [comment]
Looks great, thanks!

**Pull request merged**